graph/import.go

```diff
@@ -13,13 +13,21 @@ import (
 	"github.com/docker/docker/utils"
 )
 
+// ImageImportConfig holds configuration to import a image.
 type ImageImportConfig struct {
-	Changes         []string
-	InConfig        io.ReadCloser
-	OutStream       io.Writer
+	//Changes are the container changes written to top layer.
```

Comment:
Please add a space between "//" and the rest of your comment. (also applies to the 3 next comments)